### DISCUSSION TITLE
Password protecting pages

### DISCUSSION THREAD

**Discussion by osehgol** [mmistakes/minimal-mistakes#100] [Q&A] [answered]
Hi - me again, thanks for your help earlier on full background images (https://github.com/mmistakes/minimal-mistakes/issues/99). 

I'm trying to password protect a page on my website (go to page prompted for username/password or just password). My site is not using GitHub pages, it's setup using Digital Ocean and Git, webserve nginx). 

My Question - Is there any way to do that please, using this theme?

I have tried a couple of things (however silly :$) - 
(i) Creating a config.ru file with settings here and trying to call it in the (_layouts/page)
https://lucatironi.github.io/tutorial/2014/08/26/jekyll_static_website_continous_deployment_delivery_s3_heroku_codeship/
(ii) Adding .htaccess/.htpasswd file in the page folder
http://www.htaccesstools.com/articles/password-protection/

Thanks a lot :)

**Comment by yihangho** [comment, marked as answer]
Since you're using nginx, your best bet is to setup Basic Auth on nginx. See this [guide by DO](https://www.digitalocean.com/community/tutorials/how-to-set-up-http-authentication-with-nginx-on-ubuntu-12-10).

**Comment by mmistakes** [comment]
Thanks for jumping in and helping @yihangho
